Resources/doc/installtion.rst

```diff
@@ -0,0 +1,82 @@
+Installation
+============
+
+This chapter assumes you have Composer installed globally, as explained
+in the `installation chapter`_ of the Composer documentation.
+
+.. note::
+
+   The ODM requires the PHP 7 MongoDB driver (`mongodb`).

```

Comment:
Calling it the "PHP 7 MongoDB driver" is wrong: it also supports PHP 5. It's ODM itself that requires PHP 7.

Reply:
Changed the wording and added a link